src/NUnitFramework/tests/Attributes/TestCaseSourceTests.cs

```diff
@@ -353,6 +369,13 @@ public void Issue1337SingleMemberArrayAsArgument(string[] args)
             new int[] { 12, 6, 2 }
         };
 
+        static object[] ComplexArrayBasedTestInput = new[]
```

Comment:
I don't think that this is used anywhere.

Reply:
Fixed.